image-loader/app/controllers/ImageLoaderController.scala

```diff
@@ -56,15 +53,54 @@ class ImageLoaderController(auth: Authentication,
     quarantineUploader.map(_.quarantineFile(uploadRequest)).getOrElse(uploader.storeFile(uploadRequest))
   }
 
-  def loadImage(uploadedBy: Option[String], identifiers: Option[String], uploadTime: Option[String], filename: Option[String]): Action[DigestedFile] =  {
+  private val metadataIdentifierPrefix = "x-amz-meta-identifier!"
+  def reloadImage(uploadedBy: Option[String], identifier: String): Action[DigestedFile] = {
+    // Construct logging markers
+    implicit val context: RequestLoggingContext = RequestLoggingContext(
+      initialMarkers = Map(
+        "requestType" -> "load-image",
+        "identifiers" -> identifier,
+        "loader" -> "reloadImage"
+      )
+    )
+    // set up local file
+    val tempFile = createTempFile("s3reload")
+    // get file from s3 and write to local file
+    val parsedBody = DigestBodyParser.empty(tempFile, identifier)
+    auth.async(parsedBody) { req =>
+      for {
+        meta <- downloader.fetchImage(identifier, tempFile)
+        digestedFile = DigestedFile(tempFile, identifier)
+        // get original upload time etc from s3 meta
+        uploadedBy = meta.get("x-amz-meta-uploaded_by").getOrElse("Unknown")
+        uploadTime = meta.get("x-amz-meta-upload_time")
+        filename = meta.get("x-amz-meta-file_name")
+        // Add to logging markers
+        widerContext = context.copy(initialMarkers = context.initialMarkers + (
+          "uploadedBy" -> uploadedBy,
```

Comment:
this is shadowed from the args on 75, I think the logs might best have the uploadedBy from the function args?